webapp-client/src/component/Item.js

```diff
@@ -0,0 +1,35 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export default class Item extends React.Component {
+  formatSourceLink(link) {
+    if (link) {
+      return <a href={link}>Source</a>;
+    }
+    return '';
+  }
+
+  render() {
+    const { item } = this.props;
```

Comment:
Not sure this is that useful for a single variable

const item = this.props.item

Reply:
Oh that makes sense